ci/run-check.sh

```diff
@@ -64,6 +64,12 @@ case $TARGET in
       ./try-runtime \
       --runtime target/release/wbuild/centrifuge-runtime/centrifuge_runtime.wasm \
       on-runtime-upgrade live --uri wss://fullnode.centrifuge.io:443
+    elif [ "$1" == "demo" ]; then
+      echo "Running try-runtime for demo env"
+      RUST_LOG=runtime=trace,try-runtime::cli=trace,executor=trace \
+      ./try-runtime \
+      --runtime target/release/wbuild/development-runtime/development.wasm \
+      on-runtime-upgrade live --uri  wss://node-7118620155331796992.gx.onfinality.io/ws?apikey=00538f2d-6297-44e3-8812-4b9d579524b2
```

Comment:
Why that uri? Could it be `wss://fullnode.demo.k-f.dev`?

Reply:
fullnode.demo.k-fdev is hevily throttled so we need to use the nodes serving subquery which are dedicated and have less restrictions